metricbeat/module/zookeeper/server/_meta/data.json

```diff
@@ -0,0 +1,45 @@
+{
+    "@timestamp": "2017-10-12T08:05:34.853Z",
+    "agent": {
+        "hostname": "host.example.com",
+        "name": "host.example.com"
+    },
+    "event": {
+        "dataset": "zookeeper.server",
+        "duration": 115000,
+        "module": "zookeeper"
+    },
+    "metricset": {
+        "name": "server"
+    },
+    "service": {
+        "address": "localhost:2181",
+        "type": "zookeeper"
+    },
+    "zookeeper": {
+        "server": {
+            "connections": 1,
+            "latency": {
+                "avg": 0,
+                "max": 0,
+                "min": 0
+            },
+            "mode": "standalone",
+            "node_count": 4,
+            "outstanding": 0,
+            "received": 48,
+            "sent": 47,
+            "version": {
+                "date": "06/29/2018 04:05 GMT",
+                "id": "3.4.13-2d71af4dbe22557fda74f9a9b4309b15a7487f03"
```

Comment:
The service version should go under `service.version` as it is an ECS field. Interesting to see that you extraced the date here. @webmat Thoughts?

Reply:
Version has been moved to `service.version` and I have left the date there as `version_date`. Maybe I should simply remove it as it's something that it's not going to change and it provides not much value?

Reply:
If you easily can skip the date then let's not add it to the event. Not sure how it would be used.

Reply:
FWIW - I don't feel there's a use-case for `version_date`. It's not breaking BC if we introduce it in the future, so we might as well drop it (at least for now).